test/integration/epp/domain/create/auction_idn_test.rb

```diff
@@ -0,0 +1,282 @@
+# encoding: UTF-8
+require 'test_helper'
+
+class EppDomainCreateAuctionIdnTest < ApplicationIntegrationTest
+  def setup
+    super
+
+    @idn_auction = auctions(:idn)
+    Domain.release_to_auction = true
+  end
+
+  def teardown
+    super
+
+    Domain.release_to_auction = false
+  end
+
+  def test_registers_domain_with_ascii_idn_cannot_be_registered_without_registration_code
```

Comment:
I don't understand this. Forgot to delete `registers`?

Reply:
Copy and paste error